.github/workflows/posix.yml

```diff
@@ -49,6 +49,12 @@ jobs:
             lang: "zh_CN.utf8"
             lc_all: "zh_CN.utf8"
             name: "Locale: zh_CN.utf8"
+          - env_file: actions-38.yaml
```

Comment:
can we use this one and kill the one above?

Reply:
We can... when it works :)
If it were working, the test suite should fail because I added a test that is failing.

Right now the following message pops in the runner (as a warning not an error, so it still shows success)

> /usr/bin/bash: warning: setlocale: LC_ALL: cannot change locale (zh_CN.gb2312)

https://github.com/pandas-dev/pandas/runs/5589236201?check_suite_focus=true#step:10:37

So the locale is not set, and the tests pass without problem.
I have no idea how to make that set locale command work with this specific encoding. I tried both upper and lower caps. Any idea ? I'll try to dig into the issues and PRs history in the meantime..

Reply:
same behaviour for locale `zh_CN` (without the ".utf-8" suffix)

https://github.com/pandas-dev/pandas/runs/5602164839?check_suite_focus=true#step:10:37

Reply:
I'll now try this https://github.com/actions/virtual-environments/issues/762#issuecomment-618507288